### ISSUE TITLE
Issue (?) with Polynomials4ML

### ISSUE THREAD

**Issue by wcwitt** [ACEsuit/ACEpotentials.jl#130]
For the last few weeks, new installs of ACE1pack

```
julia -e 'using Pkg; Pkg.activate("."); Pkg.add("ACE1pack")'
```
suggest an issue with Polynomials4ML
```
Precompiling project...
  ✗ Polynomials4ML
  ? ACE1pack
  0 dependencies successfully precompiled in 7 seconds. 268 already precompiled.
  1 dependency failed but may be precompilable after restarting julia
  1 dependency errored. To see a full report either run `import Pkg; Pkg.precompile()` or load the package
```

It doesn't seem to be an actual problem, but is off-putting for new users. Does anyone understand what is happening? @cortner @tjjarvinen

**Comment by tjjarvinen** [comment]
It seems to mean that Polynomials4ML does not support precompile. This is normal and depends on the way Polynomials4ML has been implemented.

**Comment by wcwitt** [comment]
> 1 dependency failed but may be precompilable after restarting julia
  1 dependency errored. To see a full report either run `import Pkg; Pkg.precompile()` or load the package

So are you saying only the first line here refers to Polynomials4ML? What about the second line - that's the one that troubles me most? I wasn't able learn much from the "full report".

**Comment by tjjarvinen** [comment]
Yes that is precompile failing. The package should still work just fine though.

The main point with precompile is faster startup. [Here](https://julialang.org/blog/2021/01/precompile_tutorial/) is a tutorial on precompile in general.

I personally would not be worrying this. But we can also change the code, so that the error should not pop up, if that bothers people. Instructions for that are [here](https://docs.julialang.org/en/v1/manual/modules/#Module-initialization-and-precompilation). The easy solution is to add `__precompile__(false)` to Polynomials4ML.

**Comment by cortner** [comment]
What could be a reason for not precompiling?

**Comment by wcwitt** [comment]
I would prefer to do something about this if possible, because a new user came to me asking why their installation failed, which seems reasonable on their part

**Comment by cortner** [comment]
I tried the `import Pkg; Pkg.precompile()` but got no additional information. Also P4ML on its own precomplies perfectly fine. I don't understand the issue TBH.

**Comment by cortner** [comment]
nevermind, I spoke too early. Made progress. The latest P4ML is incompatible with ACE1x. This is our problem.

**Comment by cortner** [comment]
fixed in `ACE1x@0.1.2` (I didn't make a PR since it only required relaxing the version requirement for `RepLieGroups.jl`)
